src/collectors/common-contexts/system.processes.h

```diff
@@ -0,0 +1,115 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+#ifndef NETDATA_SYSTEM_PROCESSES_H
+#define NETDATA_SYSTEM_PROCESSES_H
+
+#include "common-contexts.h"
+
+#define _system_process_chart() \
+    rrdset_create_localhost( \
+        "system" \
+        , "processes" \
+        , NULL  \
+        , "processes" \
+        , NULL \
+        , "System Processes" \
+        , "processes" \
+        , _COMMON_PLUGIN_NAME \
+        , _COMMON_PLUGIN_MODULE_NAME \
+        , NETDATA_CHART_PRIO_SYSTEM_PROCESSES \
+        , update_every \
+        , RRDSET_TYPE_LINE \
+        )
+
+#if defined(COMPILED_FOR_WINDOWS)
+static inline void common_system_processes(uint64_t running, int update_every) {
+    static RRDSET *st_processes = NULL;
+    static RRDDIM *rd_running = NULL;
+
+    if(unlikely(!st_processes)) {
+        st_processes = _system_process_chart();
+
+        rd_running = rrddim_add(st_processes, "running", NULL, 1, 1, RRD_ALGORITHM_ABSOLUTE);
+    }
+
+    rrddim_set_by_pointer(st_processes, rd_running, running);
+    rrdset_done(st_processes);
+}
+
+// EBPF COUNTER PART
+static inline void common_system_threads(uint64_t threads, int update_every) {
+    static RRDSET *st_processes = NULL;
+    static RRDDIM *rd_threads = NULL;
+
+    if(unlikely(!st_processes)) {
+        st_processes = _system_process_chart();
+
+        rd_threads = rrddim_add(st_processes, "threads", NULL, 1, 1, RRD_ALGORITHM_ABSOLUTE);
+    }
+
+    rrddim_set_by_pointer(st_processes, rd_threads, threads);
+    rrdset_done(st_processes);
+}
+#endif
+
+#if defined(COMPILED_FOR_LINUX)
+static inline void common_system_processes(uint64_t running, uint64_t blocked, int update_every) {
+    static RRDSET *st_processes = NULL;
+    static RRDDIM *rd_running = NULL;
+    static RRDDIM *rd_blocked = NULL;
+
+    if(unlikely(!st_processes)) {
+        st_processes = rrdset_create_localhost(

```

Comment:
This can also be `st_processes = _system_process_chart();` now

Reply:
There was a mistake while I was adapting code for the newest pattern, it was already fixed. Thanks!